pkgs/top-level/all-packages.nix

```diff
@@ -26382,4 +26382,5 @@ in
 
   btcdeb = callPackage ../applications/blockchains/btcdeb {};
 
+  ifmetric = callPackage ../os-specific/linux/ifmetric {};
```

Comment:
please attempt to sort, `all-packages` is by no means perfect, but at least we can try to not continue the trend